actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -91,12 +91,26 @@ module DateHelper
       #   distance_of_time_in_words(from_time, from_time + 50.minutes, scope: 'datetime.distance_in_words.short') # => "an hour"
       #   distance_of_time_in_words(from_time, from_time + 3.hours, scope: 'datetime.distance_in_words.short')    # => "3 hours"
       def distance_of_time_in_words(from_time, to_time = 0, options = {})
+        if from_time.is_a?(Numeric)
+          from_time = Time.at(from_time)
+        elsif from_time.respond_to?(:to_time)
+          from_time = from_time.to_time
+        else
+          raise ArgumentError, "first argument must be convertible to_time"
+        end
+
+        if to_time.is_a?(Numeric)
+          to_time = Time.at(to_time)
+        elsif to_time.respond_to?(:to_time)
+          to_time = to_time.to_time
+        else
+          raise ArgumentError, "second argument must be convertible to_time"
+        end
```

Comment:
I wasn't absolutely certain, so I was able to confirm that methods included from a module have access to private interfaces (I suppose they're also included?)

```ruby
module Lol
  def wat
    haha
  end
  private
  def haha
    "OK"
  end
end

class Giggle
  include Lol
end

Giggle.new
=> #<Giggle:0x007ff69e0a1738>
Giggle.new.wat
=> "OK"
```